### ISSUE TITLE
`Undefined index: handle` when saving blueprint that has a field without a handle

### ISSUE THREAD

**Issue by notnek** [statamic/cms#3773]
## Bug Description

If you clear out a field's generated handle and try to save the blueprint, an exception is thrown on `Fields/Fields.php:193` because `handle` key is missing. I know that this happening is edge case since the handle is generated for you, but I've had clients delete it and try to save…

## How to Reproduce 

1. Create or edit a blueprint
2. Create Field
3. Delete handle field's value
4. "Finish" 
5. "Save" blueprint

## Extra Detail
```json
{
  "handle": "test_blueprint",
  "hidden": false,
  "sections": [
    {
      "handle": "main",
      "display": "Main",
      "fields": [
        {
          "_id": "kox40b30",
          "type": "inline",
          "fieldtype": "text",
          "icon": "text",
          "handle": null,
          "config": {
            "placeholder": null,
            "input_type": "text",
            "character_limit": null,
            "prepend": null,
            "append": null,
            "antlers": false,
            "display": "Test field",
            "type": "text",
            "icon": "text",
            "instructions": null,
            "localizable": false,
            "width": 100,
            "listable": "hidden",
            "validate": [
              "required"
            ]
          }
        }
      ],
      "_id": "main"
    }
  ]
}
```

```
ErrorException · Undefined index: handle
```

```php
// Otherwise, the field has been configured inline.
return $this->newField($config['handle'], $config['field']);
```

## Environment

Statamic 3.1.14 Pro
Laravel 7.30.4
PHP 7.3.28
No addons installed

**Comment by notnek** [comment]
For question #1, 🤷🏻 , clients be clients. For question #2, they build forms which uses the blueprint editor.

**Comment by duncanmcclean** [comment]
I've just tested this issue on the latest version - a validation error will pop up if there's a missing handle when saving a field's settings.

Fixed in 87b0b04.
